Documentation/source/conf.py

```diff
@@ -81,7 +82,10 @@
         }
     ],
     "footer_start": ["crown-copyright"],
-    "footer_center": ["sphinx-version"],
+    "footer_center": [
+        "sphinx-version",
+        f"Updated: {datetime.datetime.now().strftime('%Y-%m-%d %H:%M')}",
```

Comment:
I don't think this is correct: the list entries should reference templates file names rather than literal strings.  I think you need to add:

```python
html_last_updated_fmt = '%Y-%m-%d %H:%M'
```

Before this section and then change these lines to:

```suggestion
        "sphinx-version",
        "last-updated",
```